### PR TITLE
Virtual Nursing Assistant Log Update Card for Automated Daily Rounds

### LINKED ISSUE #4587: Log update card size is not persistent
**Describe the bug**
Log update card size is not persistent

**To Reproduce**

1. Go to https://care.coronasafe.in/facility/d02ef2cb-b5b2-4baf-bc9a-fac348893b00/patient/f0e482ca-ffd7-4bed-87e5-2539a6f20bf8/consultation/6f0624b3-afeb-44ff-b2f0-55bacc319afa

**Expected behavior**
Card size should be persistent regardless of the content of the log update

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### LINKED ISSUE #4718: Mark Rounds by Virtual Nursing Assistant
Log Updates with RoundType = 300 should be marked as "Created by Virtual Nursing Assistant" in the Consultation Log History

### LINKED ISSUE #4963: Replace Material UI components in `src/Components/Facility/Consultations/DailyRoundsList.tsx`:
```tsx
1: import { Grid, Typography } from "@material-ui/core";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [x] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [x] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [x] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #4718
- Fixes #4587 
- Fixes #4963

https://deploy-preview-4726--care-net.netlify.app/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/4e339b9d-1547-4f2f-9b80-64b378d9d835/consultation/781e00b4-af6a-4454-a729-52ae674d6948

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by rithviknishad** [comment]
@mathew-alex could you connect a monitor for the automated daily rounds to patient Annie Philip?

**Comment by nihal467** [comment]
@mathew-alex what is the update on this ?

**Comment by mathew-alex** [comment]
> @mathew-alex what is the update on this?

I am working with @sainak to set up dummy vitals in the dev server. I will have an update in the next couple of days.

**Comment by nihal467** [comment]
@mathew-alex what is the update on this PR

**Comment by mathew-alex** [comment]
> @mathew-alex what is the update on this PR

vitals are set up. but the automated daily round is not working. @khavinshankar is looking into it.

**Comment by nihal467** [comment]
@khavinshankar what is the status on this PR

**Comment by rithviknishad** [comment]
@nihal467 @khavinshankar ready

**Comment by nihal467** [comment]
@rithviknishad 
image

the card is not responsive in mobile view

**Comment by rithviknishad** [comment]
@nihal467 fixed 
image

File: src/Components/Facility/Consultations/DailyRounds/DefaultLogUpdateCard.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  const by = props.consultationData.assigned_to_object || round.created_by;
+
+  return (
+    <div
+      className={`p-4 flex flex-col gap-4 w-full rounded-lg shadow ${
+        telemedicine_doctor_update ? "bg-purple-200" : "bg-white"
+      }`}
+    >
+      <div className="flex flex-col gap-1 md:flex-row items-start md:items-center justify-between">
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/DailyRounds/DefaultLogUpdateCard.tsx]:
```suggestion
      <div className="flex flex-col gap-1 items-start">
```

File: src/Components/Facility/Consultations/DailyRounds/VirtualNursingAssistantLogUpdateCard.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    getDeepDiff(
+      extractVirtualNursingAssistantFields(props.round),
+      extractVirtualNursingAssistantFields(props.previousRound)
+    );
+
+  const diffKeys = Object.keys(diff);
+
+  return (
+    <div className="p-4 flex flex-col gap-4 w-full rounded-lg bg-white shadow shadow-primary-500/20 border border-green-300">
+      <div className="flex flex-col gap-1 md:flex-row items-start md:items-center justify-between">
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/DailyRounds/VirtualNursingAssistantLogUpdateCard.tsx]:
```suggestion
      <div className="flex flex-col gap-1 items-start">
```

**Comment by nihal467** [comment]
LGTM , https://deploy-preview-4726--care-net.netlify.app/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/8db67f09-fa73-43e7-8728-d227e6397e79/consultation/6ea4b2d2-12c1-44eb-8c64-c41ec9ced4eb
@rithviknishad great work
@khavinshankar you can review it and merge it

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**